packages/graph/Makefile.toml

```diff
@@ -23,10 +23,9 @@ cwd = "${REPO_ROOT}/packages/graph/migrations"
 [tasks.migrate-up]
 category = "Deploy"
 description = "Runs all migrations"
-extend = "yarn"
-args = ["graph:migrate", "up"]
```

Comment:
Is there even a purpose of us keeping this `graph:migrate` command in yarn then?

I know @benwerner01 found it somewhat confusing that the yarn had two different migration commands that behave differently. (And I mean I think we should seek to unify them at some point anyway, but in the meantime perhaps it'd be best to remove one of the `yarn` ones and leave this as a manual docker thing expressed through Cargo make, Rust tooling for Rust workflows)

Reply:
The Dockerfile will use the `yarn` command, so fully remove it isn't possible. An alternative would be to simply remove the `cargo make` commands for `migrate-up` and `recreate-db` and run the yarn commands directly, however, we need to pass the .env file path manually then.

The easiest way is probably to just keep them for now.